hack/generate-monitoring-docs.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+echo "> Generating monitoring docs..."
```

Comment:
Nit: for consistency with the rest of the output of `make generate`
```suggestion
echo "Generating monitoring docs"
```

Reply:
Based on https://github.com/gardener/gardener/blob/848c7a187d879fa97a4246a1ba5a8ba41425502f/hack/generate.sh#L19, I am using 
```bash
echo "> Generate monitoring docs"
```